### ISSUE TITLE
Missing doc/fribidi_log2vis.3 causes installation failure

### ISSUE THREAD

**Issue by lantw44** [fribidi/fribidi#129]
This happens when building from git.
```
ninja -j4 install
[1/2] Generating man pages with a custom command.
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_shape_arabic.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_par_direction.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_par_embedding_levels_ex.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_reorder_line.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_bidi_type.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_bidi_types.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_bidi_type_name.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_charset_to_unicode.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_unicode_to_charset.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_parse_charset.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_debug_status.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_set_debug.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_mirroring_status.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_set_mirroring.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_reorder_nsm_status.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_set_reorder_nsm.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_log2vis_get_embedding_levels.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_type.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_type_internal.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_par_embedding_levels.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_join_arabic.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_joining_type.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_joining_types.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_joining_type_name.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_mirror_char.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_shape_mirroring.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_bracket.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_get_bracket_types.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_shape.3
generating: /home/lantw44/gnome/build/fribidi/doc/fribidi_remove_bidi_marks.3
[1/2] Installing files.
Installing gen.tab/fribidi-unicode-version.h to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/include/fribidi
Installing lib/libfribidi.so.0.4.0 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/lib
Installing bin/fribidi to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/bin
Installing doc/fribidi_charset_to_unicode.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_debug_status.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_bidi_type.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_bidi_type_name.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_bidi_types.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_bracket.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_bracket_types.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_joining_type.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_joining_type_name.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_joining_types.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_mirror_char.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_par_direction.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_par_embedding_levels.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_par_embedding_levels_ex.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_type.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_get_type_internal.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Installing doc/fribidi_join_arabic.3 to /home/lantw44/gnome/devinstall/_jhbuild/root-fribidi/home/lantw44/gnome/devinstall/share/man/man3
Traceback (most recent call last):
  File "/home/lantw44/gnome/devinstall/lib/python3.6/site-packages/mesonbuild/mesonmain.py", line 129, in run
    return options.run_func(options)
  File "/home/lantw44/gnome/devinstall/lib/python3.6/site-packages/mesonbuild/minstall.py", line 521, in run
    installer.do_install(datafilename)
  File "/home/lantw44/gnome/devinstall/lib/python3.6/site-packages/mesonbuild/minstall.py", line 343, in do_install
    self.install_targets(d)
  File "/home/lantw44/gnome/devinstall/lib/python3.6/site-packages/mesonbuild/minstall.py", line 436, in install_targets
    raise RuntimeError('File {!r} could not be found'.format(t.fname))
RuntimeError: File 'doc/fribidi_log2vis.3' could not be found
FAILED: meson-install 
/home/lantw44/gnome/devinstall/bin/meson install --no-rebuild
ninja: build stopped: subcommand failed.
```

**Comment by atsampson** [comment]
I'm seeing this with c2man 577ed4095383ef5284225d45709e6b5f0598a064 as well - c2man doesn't see the definition of `fribidi_log2vis` at all! Sticking a dummy comment after the definition seems to be a workaround:

```
--- tmp/v1.0.8/lib/fribidi.h
+++ work/v1.0.8/lib/fribidi.h
@@ -110,6 +109,8 @@
   FriBidiLevel *embedding_levels       /* output list of embedding levels */
 );
 
+/* hey, c2man! */
+
 #ifdef FRIBIDI_NO_DEPRECATED
 #else
 # include "fribidi-deprecated.h"

```

**Comment by dov** [comment]
Fixed in commit 1e48733 .
